engine_test.go

```diff
@@ -4311,6 +4409,20 @@ func allTestTables(t *testing.T, numPartitions int) map[string]*memory.Table {
 		{Name: "bl", Type: sql.Blob, Source: "typestable", Nullable: true},
 	}, numPartitions)
 
+	tables["stringandtable"] = memory.NewPartitionedTable("stringandtable", sql.Schema{
+		{Name: "i", Type: sql.Int64, Source: "stringandtable", Nullable: false},
+		{Name: "v", Type: sql.Text, Source: "stringandtable", Nullable: false},
+	}, numPartitions)
+
+	insertRows(
```

Comment:
You need nulls for a full test case. Ideally you should have a full truth table with every combination of true, false, null